### PR TITLE
zoolander(bugfix): adjusted ticker string so that it won't jitter around

### PR DESCRIPTION
**Pull request by runrog**
https://jira.rax.io/browse/RSWEB-6926

### CODE REVIEW

**Comment by WidgetsBurritos** [comment]
+1 looks okay for me.

File: styleguide/_themes/derek/scss/components/ticker.scss [hunk 1]
```diff
@@ -55,9 +55,10 @@ $ticker-span-bg: rgba(200, 0, 0, .5);
   font-size: 29px;
   font-weight: 700;
   letter-spacing: 6px;
-  margin: 30px;
+  margin: 30px auto;
   padding: 15px;
   text-align: left;
+  width: 265px;
```
Comment (amberRrucker) [inline styleguide/_themes/derek/scss/components/ticker.scss]: i still dont love the idea of used a fixed pixel width and would prefer to use the make-(size)-col to vary the widths and break points. @bighappyface and @SyntaxStacks thoughts?

Comment (SyntaxStacks) [inline styleguide/_themes/derek/scss/components/ticker.scss]: I didn't see this! Yeah i'd rather have the columns in place.

Comment (amberRrucker) [inline styleguide/_themes/derek/scss/components/ticker.scss]: REVERT REVERT! haha. I thought we were doing 2 plus 1s for every release now.

**Pull request merged**